frontend/src/bundles/common/hooks/use-height/use-height.hook.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+
+const BREAKPOINT = {
+    MEDIUM: 680,
+} as const;
+
+const useHeight = (): boolean => {
+    const [isHeight, setIsHeight] = useState(false);
+
+    const responsiveHighness = (): void => {
+        if (window.innerHeight <= BREAKPOINT.MEDIUM) {
+            setIsHeight(true);
+        } else {
+            setIsHeight(false);
+        }
+    };
+
+    useEffect(() => {
+        window.addEventListener('resize', responsiveHighness);
+        responsiveHighness();
+    }, [isHeight]);
+
+    return isHeight;
+};
```

Comment:
How should this scale in the future? What if we will different breakpoint? It might be worth exposing the current breakpoint instead of just a boolean flag or at least give ability to configure this hook. Also, don't forget to clean up the event listener.